tools/buildsys/src/builder.rs

```diff
@@ -93,6 +93,9 @@ impl VariantBuilder {
         let version_image = getenv("BUILDSYS_VERSION_IMAGE")?;
         let version_build = getenv("BUILDSYS_VERSION_BUILD")?;
         let output_dir: PathBuf = getenv("BUILDSYS_OUTPUT_DIR")?.into();
+        // We expect users' PRETTY_NAME values to contain spaces for things like "Bottlerocket OS"
+        // and so we need to transform them the same way as PACKAGES above.
+        let pretty_name = getenv("BUILDSYS_PRETTY_NAME")?.replace(' ', "|");
```

Comment:
I assume this means that `|` ends up being a reserved character that you can't use in the pretty name.  Is that correct?  Should we document that in the readme?

Reply:
A pipe would become a space. I've never seen that used before in an OS name. I think it'd be cluttering the docs... no?

Reply:
Doesn't seem like it would be common, but it wouldn't surprise me to see someone want to use it.  I do think we should document it somewhere, but we could put it somewhere other than the readme if you think that's too much clutter.

Reply:
I'm going to think about alternatives in the code so this isn't required.

Reply:
Please see the new second commit and updated testing, I removed this character restriction.